pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -7899,11 +7899,11 @@ TRANSLATE_NOOP("FullscreenUI", "BIOS Settings");
 TRANSLATE_NOOP("FullscreenUI", "Emulation Settings");
 TRANSLATE_NOOP("FullscreenUI", "Graphics Settings");
 TRANSLATE_NOOP("FullscreenUI", "Audio Settings");
+TRANSLATE_NOOP("FullscreenUI", "Folder Settings");
+TRANSLATE_NOOP("FullscreenUI", "Achievements Settings");
```

Comment:
I don't think the order matters here, but these two are after the memcard settings elsewhere.